backend/sessionManagementRoutes.js

```diff
@@ -115,7 +119,7 @@ const configureRoutes = ({ app, tokenRefresher, tokenVerifier, homeLink }) => {
         logger.info(`Auth failure due to ${JSON.stringify(info)}`)
         return res.redirect('/autherror')
       }
-      req.logIn(user, (err2) => {
+      req.login(user, (err2) => {
```

Comment:
these are override of passport function so no changed to sign in.  Can be called with either `logIn` or `login` - see https://github.com/jaredhanson/passport/blob/master/lib/http/request.js#L31